### PR TITLE
Zoneinfo

### PR DESCRIPTION
**Pull request by S00185883**
Screenshot 2021-08-20 at 15 12 56
Screenshot 2021-08-20 at 15 13 04

### CODE REVIEW

File: src/components/AllModals/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+function AllModals() {
+
+    return (
+        <>
+            <Modal body={reentryText} id={"modal1"}>
+          {" "}
+        </Modal>
+        {/* <Modal
```
Comment (eim-ostk) [inline src/components/AllModals/index.js:14]: Uh ideally tidy up some of this crap I threw together quickly 🤣

File: src/componenttext/ZoneInfoText.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+// import swift from "./images/swift.png";
+// import "materialize-css";
+// import { Row, Col } from "react-materialize";
+// import "materialize-css/dist/css/materialize.min.css";
+// import moment from "moment";
+// import axios from "axios";
+
+// function ZoneInfoText( { selectedSlot } ) {
+//  const zoneInfo = (() => {
+//     if ( selectedSlot.roomId === 1 ) {
```
Comment (eim-ostk) [inline src/componenttext/ZoneInfoText.js:13]: If this is no longer needed, I'd delete it

File: src/index.js [hunk 3]
```diff
@@ -4,11 +4,13 @@ import "./index.css";
 import reportWebVitals from "./reportWebVitals";
 import CalendarWrapper from "./components/home/CalendarWrapper";
 import Header from "./components/header/Header";
+import AllModals from "./components/AllModals";
 
 ReactDOM.render(
   <React.StrictMode>
-    <Header />
+    {/* <Header /> */}
```
Comment (eim-ostk) [inline src/index.js:11]: If no longer needed, I'd delete it

**Review by eim-ostk — CHANGES_REQUESTED** [review]
Looks well, just a few comments

**Review by S00189591 — DISMISSED** [review]
All the checks failed, not sure what that means though?

**Comment by Clancinio** [comment]
Always a good idea to run yarn test regularly to make sure tests are passing

File: src/components/flyout/footer/Footer.js [hunk 4]
```diff
@@ -78,31 +199,27 @@ function Footer({
...
+      </a>
+      <a href="#!" className="modal-close waves-effect waves-light btn-large">
+        Go Back
+      </a>
+      <br></br>
+      <br></br>
     </div>
   );
-
+  // eslint-disable-next-line
```
Comment (Clancinio) [inline src/components/flyout/footer/Footer.js:216]: Why does this comment keep appearing??

File: src/componenttext/ZoneInfoText.js (hunk 2 above)
Comment (S00185883) [reply src/componenttext/ZoneInfoText.js:13]: Were going to try and move the modals into that all modal file so its staying until we can play around more with it

**Pull request merged**